### DISCUSSION TITLE
how to hide git:(master)

### DISCUSSION THREAD

**Discussion by harryinnotech** [gitextensions/gitextensions#11213] [Q&A]
please my vscode terminal with wsl integration is showing me 
harry 
git:(master) 
is there any way I can hid the git:(master) but still be able to commit changes to my git repository.

**Comment by vbjay** [comment]
https://letmegooglethat.com/?q=Vs+code+prompt+setting

This is the gitextensions gui project. Yes we have an extension to drive ui in vs code but you are looking in how to manage the prompt in vs code. So wrong repo.
